verticapy/machine_learning/feature_extraction/text.py

```diff
@@ -0,0 +1,197 @@
+from verticapy._typing import SQLRelation
```

Comment:
please add OpenText header string. It is in all our files. Also format your code using the latest version of black.